manta-js/package/src/privateWallet.ts

```diff
@@ -421,6 +424,60 @@ export class MantaPrivateWallet implements IMantaPrivateWallet {
     }
   }
 
+  /// Gets the storage for the specific Network Signer and updates the IndexDB storage
+  /// Returns true if successful, false if not
+  /// TODO: do not rewrite whole storage on updates (after prototype)
+  async updateStorage(): Promise<boolean> {
+    try {
+      await this.waitForWallet();
+      this.walletIsBusy = true;
```

Comment:
I think storing utxos to storage should be asynchronous, it should not block user interaction. even if storage fails

Reply:
Yeah should be fine since we are just reading the signer storage. Will update